src/pages/ContentfulPage.vue

```diff
@@ -348,6 +383,28 @@ export default {
 				this.pageBackgroundColor = pageLayout.pageBackgroundColor ?? '';
 				this.pageFrame = getPageFrameFromType(page.pageType);
 				this.contentGroups = getContentGroups(pageData);
+
+				if (page.key === 'hp/crowdfund-for-good') {
```

Comment:
I'm not sure if moving this channel-specific logic to the contentful page is the most appropriate choice. The contentful page runs a bunch of different page types, so having this type of specific logic here is a little unusual. Have we discussed putting this type of page specific logic in this wrapper page component? If not, it might make more sense to extract this logic into a util and call the util in the two files that need this channel/loan data.

Reply:
I agree, we should not be including any component specific or loan related configuration or data fetchs in this file. We Also should not be calling any utils for any targted content groups.

This file needs to remain totally generic with the exeception of the redirect controls we have in place already.